tests/Unit/Scraper/Listeners/UpdateDatasetTest.php

```diff
@@ -101,7 +99,7 @@ public function whenDatasetDoesNotExistAndTheDatasetsLimitHasNotBeenReachedItSho
     public function whenDatasetDoesNotExistAndTheDatasetsLimitHasReachedItShouldDeleteTheExcess(): void
     {
         $type = ':type:';
-        factory(ScrapedDataset::class, UpdateDataset::DATASET_AMOUNT_LIMIT + 10)->create([
+        ScrapedDataset::factory()->count(UpdateDataset::DATASET_AMOUNT_LIMIT)->create([
```

Comment:
Why this change in behavior? Is the test testing what it is supposed after this change?

Reply:
The test should fulfill the purpose described in the test name. In this case `DeleteTheExcess`.